### PR TITLE
disabled the auto assign bot

### PR DESCRIPTION
**Pull request by gaaclarke**
## Description

Diabled the auto assign bot

## Tests

I added the following tests:

*Replace this with a list of the tests that you added as part of this PR. A
change in behaviour with no test covering it will likely get reverted
accidentally sooner or later. PRs must include tests for all
changed/updated/fixed behaviors. See [testing the engine] for instructions on
writing and running engine tests.*

## Checklist

Before you create this PR confirm that it meets all requirements listed below by checking the relevant checkboxes (`[x]`). This will ensure a smooth and quick review process.

- [x] I read the [contributor guide] and followed the process outlined there for submitting PRs.
- [x] I signed the [CLA].
- [x] I read and followed the [C++, Objective-C, Java style guides] for the engine.
- [x] I read the [tree hygiene] wiki page, which explains my responsibilities.
- [x] I updated/added relevant documentation.
- [x] All existing and new tests are passing.
- [x] I am willing to follow-up on review comments in a timely manner.

## Breaking Change

Did any tests fail when you ran them? Please read [handling breaking changes].

- [x] No, no existing tests failed, so this is *not* a breaking change.
- [ ] Yes, this is a breaking change. *If not, delete the remainder of this section.*
   - [ ] I wrote a design doc: https://flutter.dev/go/template *Replace this with a link to your design doc's short link*
   - [ ] I got input from the developer relations team, specifically from: *Replace with the names of who gave advice*
   - [ ] I wrote a migration guide: https://flutter.dev/go/breaking-changes-template *Replace this with a link to a pull request that adds the migration guide to https://flutter.dev/docs/release/breaking-changes*

[issue database]: https://github.com/flutter/flutter/issues
[contributor guide]: https://github.com/flutter/flutter/wiki/Tree-hygiene#overview
[testing the engine]: https://github.com/flutter/flutter/wiki/Testing-the-engine
[C++, Objective-C, Java style guides]: https://github.com/flutter/engine/blob/master/CONTRIBUTING.md#style
[CLA]: https://cla.developers.google.com/
[tree hygiene]: https://github.com/flutter/flutter/wiki/Tree-hygiene
[handling breaking changes]: https://github.com/flutter/flutter/wiki/Tree-hygiene#handling-breaking-changes

### CODE REVIEW

File: .github/auto_assign.yml [hunk 1]
```diff
@@ -2,7 +2,7 @@
 # https://github.com/kentaro-m/auto-assign/
 
 # Set to true to add reviewers to pull requests
-addReviewers: true
+addReviewers: false
```
Comment (chinmaygarde) [inline .github/auto_assign.yml:5]: Does this file do anything now? Can we just remove the whole thing?

Comment (gaaclarke) [reply .github/auto_assign.yml:5]: Zach decided to punt on landing this just yet, I'll close it out.  FWIW I thought disabling the bot via configuration files was ideal to poking people to kill it.

**Comment by gaaclarke** [comment]
Zach and I talked, punting on this PR while we consider alternatives.

**Comment by zanderso** [comment]
In particular, the original goals of this bot IIUC were to distribute reviewing load and to help spread knowledge among team members. The first goal I'm not sure we can manage, but the second one sounds achievable to me if we combine this bot with something else. We will spend a small amount of time discussing what that something else should be, and if we can't come up with anything, then we should land this PR (either as-is, or with Chinmay's suggestion).

**Comment by zanderso** [comment]
Okay. It's been a couple of weeks, and it looks like getting this in a place where it can achieve its goals will take a bit more of a concrete plan. So, let's turn this down until we can work out what that is.

**Comment by chinmaygarde** [comment]
@gaaclarke Are you going to carry this PR to completion? If not, lets close it.

**Comment by gaaclarke** [comment]
Sorry, does `So, let's turn this down until we can work out what that is.` mean we want to land it?  @zanderso @chinmaygarde  I assume so since the pr was reopened.

**Comment by chinmaygarde** [comment]
Yes. Though I prefer we just remove the [YAML file](https://github.com/flutter/engine/pull/21341/files#r493859010).

**Pull request merged**